tailwind.config.js

```diff
@@ -12,9 +12,19 @@ module.exports = {
     spacing: {
       quarter: '5px',
       half: '10px',
+      threeQuarter: '15px',
       base: '20px',
       double: '40px',
     },
+    borderRadius: {
+      default: '6px',
+    },
+    boxShadow: {
+      small: '0px 2px 4px rgba(0, 0, 0, 0.06)',
+      default: '0px 3px 6px rgba(0, 0, 0, 0.06)',
+      large: '0px 8px 16px rgba(0, 0, 0, 0.1)',
+      xlarge: '0px 15px 30px rgba(0, 0, 0, 0.2)',
+    },
```

Comment:
🚩 Worth pinging Peaks on these, he updated the box shadows to work better on dark backgrounds (unless these are from GCDS)

Reply:
These are from GCDS!